plugins/cloud-azure/src/main/java/org/elasticsearch/cloud/azure/storage/AzureStorageService.java

```diff
@@ -44,9 +44,6 @@
 
         public static final String TIMEOUT = "cloud.azure.storage.timeout";
 
-        public static final String ACCOUNT = "repositories.azure.account";
-        public static final String LOCATION_MODE = "repositories.azure.location_mode";
-        public static final String KEY = "cloud.azure.storage.key";
         public static final String CONTAINER = "repositories.azure.container";
         public static final String BASE_PATH = "repositories.azure.base_path";
```

Comment:
Agreed. I'm still wondering if we should actually support a global `base_path` setting which was the original intention (so I think this is a bug).
Or if we should only support `base_path` for individual repository.

WDYT?